Source/Request.swift

```diff
@@ -308,7 +308,12 @@ extension Request: CustomDebugStringConvertible {
                 let cookies = cookieStorage.cookies(for: url), !cookies.isEmpty
             {
                 let string = cookies.reduce("") { $0 + "\($1.name)=\($1.value);" }
-                components.append("-b \"\(string.substring(to: string.characters.index(before: string.endIndex)))\"")
+
+                #if swift(>=3.2)
+                    components.append("-b \"\(string[..<string.index(before: string.endIndex)])\"")
```

Comment:
`index(before: endIndex` gives you `endIndex - 1`. `..<endIndex` also gives you `endIndex - 1`. `..<index(before: endIndex)` would give you `endIndex - 2`. So to be equivalent to what it was before it would be `..<endIndex`, no?